homeassistant/components/zimi/entity.py

```diff
@@ -21,19 +21,19 @@ class ZimiEntity(Entity):
     _attr_should_poll = False
     _attr_has_entity_name = False
 
-    def __init__(self, entity: ControlPointDevice, api: ControlPoint) -> None:
+    def __init__(self, device: ControlPointDevice, api: ControlPoint) -> None:
         """Initialize an HA Entity which is a ZimiDevice."""
 
-        self._entity = entity
+        self._entity = device
```

Comment:
Please change the attribute name too.
```suggestion
        self._device = device
```

Reply:
See https://github.com/home-assistant/core/pull/129876/commits/bb6d3455ec2d243830e68a4ead7e6899dee433ea